WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -390,13 +390,32 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     }
 }
 
+// MARK: - ViewModel Factory
```

Comment:
I made this accessible so I could add more unit tests. I was thinking of testing the view controllers [exposed by `handleError`](https://github.com/woocommerce/woocommerce-ios/blob/72659b2215751445d98b5bc61dd226d9b4f001c4/WooCommerce/Classes/Authentication/AuthenticationManager.swift#L206-L214)  and use [the `viewModel`](https://github.com/woocommerce/woocommerce-ios/blob/72659b2215751445d98b5bc61dd226d9b4f001c4/WooCommerce/Classes/Authentication/Navigation%20Exceptions/ULErrorViewController.swift#L10-L12) to test. But I didn't want to make that public. That seems worse to me. 

Let me know if there's a better way to do this.